### DISCUSSION TITLE
How to have consistent cursor when switching to different windows

### DISCUSSION THREAD

**Discussion by alok-old** [end-4/dots-hyprland#408] [Q&A] [answered]
So I enabled The Bibata cursor from **Genome tweeks** but as expected it only appears in Wayland supported apps. I want it to appear everywhere with same size and style. Sorry if this is a dumb question.

**Comment by PharmaRacist** [comment]
add this to exec config using hyprctl `exec-once = hyprctl setcursor Bibata-Modern-Classic 24
`

**Reply by alok-old** [reply]
I just checked and it was already there, but the cursor is not applying. When I tried running `hyprctl setcursor Bibata-Modern-Classic 24` I got `ok` as a response but no change

**Reply by PharmaRacist** [reply]
this is an issue in the package of bibata try reinstalling it or change to Bibata-Modern-Ice 24

**Reply by alok-old** [reply]
I tried putting the Bibata-Modern-Ice folder in both `.icons` and `.local/share/icons/`, and then tried both `hyprctl setcursor Bibata-Modern-Ice 24` and `gsettings set org.gnome.desktop.interface cursor-theme 'Bibata-Modern-Ice'`. Am I missing something

Edit: The cursor is not applying in eg: Brave (flatpak) and also in the blank desktop (above the wallpaper).

**Reply by PharmaRacist** [reply]
exec = hyprctl setcursor Bibata-Modern-Ice 20
exec = gsettings set org.gnome.desktop.interface cursor-theme 'Bibata-Modern-Ice'
exec = gsettings set org.gnome.desktop.interface cursor-size 20

**Reply by PharmaRacist** [reply]
check the env of x cursor

**Reply by alok-old** [reply]
Can you please tell me where the env of x cursor is

**Reply by end-4** [reply, marked as answer]
this?
```ini
env = XCURSOR_THEME,Bibata-Modern-Classic
env = XCURSOR_SIZE,24
```
